Onepunch/README.md

```diff
@@ -1,18 +1,38 @@
-# Onepunch
+# Onepunch by Saitama D. Luffy
 
 ## Screenshots
 
 ![OnepunchHome](./OnepunchHome.png)
 ![OnepunchFullPlayer](./OnepunchFullPlayer.png)
 
 ## More
-This is still an alpha project intended to redesign the app as much as possible.
-But it will take time as I do this as a hobby. Hope you will like it.
 
-A theme by Saitama D. Luffy
+#### Details
 
-PS: feedbacks will be much appreciated. Specially from windows users. I can't test
-in windows.
+>&nbsp;&nbsp;&nbsp;&nbsp;A project intended to redesign the Spotify app.As you know Saitama is only a "hero for fun", this is going to take a while even though he can finish an enemy with just one punch. I hope you'll like it! Suggestions will always help me to make it look better.
```

Comment:
In general also I suggest you to avoid using `>` for emphasizing; it is not really necessary. If you REALLY need it you can use headers (with `######`)